<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rank Verify - Проверка разрядов</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" rowspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Настройки</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cb_MS">
        <property name="text">
         <string>МС</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cb_KMS">
        <property name="text">
         <string>КМС</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="cb_2r">
        <property name="text">
         <string>2 р  и ниже</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="cb_1r">
        <property name="text">
         <string>1 р</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="TextEdit">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="AboutButton">
     <property name="text">
      <string>О программе</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="HelpButton">
     <property name="text">
      <string>Помощь</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="FileNameEdit_2">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="OpenFileButton_2">
     <property name="text">
      <string>Выберите файл 
для проверки</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="FileNameEdit_1">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="OpenFileButton_1">
     <property name="text">
      <string>Выберите эталонный
 файл</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
